src/sentry/models/groupregression.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+
+
+class GroupRegression(Model):
+    """
+    Describes when a group was marked as a regression.
+    """
+
+    __include_in_export__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True)
+    # the release in which this regressed in
+    release = FlexibleForeignKey("sentry.Release")
+    datetime = models.DateTimeField(default=timezone.now, db_index=True)
```

Comment:
Let's make this `date_added` so that it's consistent with fields on other tables

Reply:
Ah okay. Copied from GroupResolution but that one appears to be the odd one out based on some random checks